examples/csv/csb43.csv.rules

```diff
@@ -0,0 +1,15 @@
+# csb43.csv.rules
+# Rules for csb2format -f csv "${CSV_FILE}" -
+# Using https://pypi.org/project/csb43/
+
+# Skip header
+skip 1
+
+# Change to your taste
+account1 assets:bank:csb43
+
+# These appear to be the useful columns for hledger
+fields _,_,_,_,_,_,currency,_,_,_,_,_,_,_,_,_,_,_,_,item1,item2,reference1,reference2,date,_,amount,_,_
```

Comment:
I think (for 1.16 and later) it is best to call columns currency1 and amount1. Plus from the csv sample it seems that you can add balance1 as well

Reply:
I'm fine with currency/amount too, they are affecting both postings after all. Your call OP.

Nice example!